components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -112,4 +117,51 @@ function AutomaticPointsTable:parseManualPoints(team, tournamentCount)
 	return manualPoints
 end
 
+function AutomaticPointsTable:queryPlacements(tournaments)
+	local queryParams = {
+		limit = 5000,
+		query = 'tournament, participant, placement, extradata'
+	}
+
+	local tournamentNames = Table.mapValues(tournaments,
+		function(tournament)
+			return tournament.name
+		end
+	)
+
+	local conditions = '[[tournament::' ..
+		table.concat(tournamentNames, ']] OR [[tournament::') ..
+		']]'

```

Comment:
Consider using #946, which will be merged today-ish.

Reply:
I think for the purpose of this module it's not really needed, however if needed we can use this as well
```suggestion
	local conditionTree = ConditionTree(BooleanOperator.any)
	Table.forEach(tournaments,
		function(tournament)
			conditionTree:add(
				ConditionNode(
					ColumnName('tournament'),
					Comparator.eq,
					tournament.name
				)
			)
		end
	)
	local conditions = conditionTree:toString()
```

Reply:
```suggestion
	local tree = ConditionTree(BooleanOperator.any)
	local columnName = ColumnName('tournament')
	Table.forEach(tournaments,
		function(t)
			tree:add(ConditionNode(columnName, Comparator.eq, t.name))
		end
	)
	local conditions = tree:toString()
```

Can be much shorter :)

Reply:
Will use your first suggestion, looks pretty good to me